### PR TITLE
fix: dockerfile npm run build

### PR DESCRIPTION
**Pull request by stefangeneralao**
Removed comment suggesting to use npm as an alternative but it's already configured by default.

### CODE REVIEW

**Review by styfle — APPROVED** [review]
I'm guessing this should say `pnpm run build` or `yarn run build` but it can probably be removed

**Comment by stefangeneralao** [comment]
@styfle Yes, exactly right. The file had the following two days ago:

```
RUN yarn build

# If using npm comment out above and use below instead
# RUN npm run build
```

File: examples/with-docker/Dockerfile [hunk 1]
```diff
@@ -29,9 +29,6 @@ COPY . .
 
 RUN npm run build
```
Comment (styfle) [inline examples/with-docker/Dockerfile:30]:
```suggestion
RUN \
  if [ -f yarn.lock ]; then yarn run build; \
  elif [ -f package-lock.json ]; then npm run build; \
  elif [ -f pnpm-lock.yaml ]; then pnpm run build; \
  else echo "Lockfile not found." && exit 1; \
  fi
```

**Review by styfle — APPROVED** [review]
I just pushed an update to add the if/else, thanks!

**Comment by stefangeneralao** [comment]
Beautiful! Thanks

**Pull request merged**